src/couch/src/couch_db.erl

```diff
@@ -1982,6 +1983,7 @@ increment_stat(#db{options = Options}, Stat, Count) when
 ->
     case lists:member(sys_db, Options) of
         true ->
+            %% TODO: we shouldn't leak resource usage just because it's a sys_db
```

Comment:
Hrmm. How we can do it? Are you planing to store `IS_SYSTEM_DB` key on process dictionary or something?

Final solution to this question is out of scope for the first PR. TODO is good enough for first version of the feature.